target/linux/ramips/image/mt7621.mk

```diff
@@ -286,6 +286,25 @@ define Device/arcadyan_we420223-99
 endef
 TARGET_DEVICES += arcadyan_we420223-99
 
+define Device/zte_mf286c3
+  $(Device/nand)
+  $(Device/uimage-lzma-loader)
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  VID_HDR_OFFSET := 2048
+  IMAGE_SIZE := 41984k
+  KERNEL_SIZE := 5120k
+  UBINIZE_OPTS := -E 5
+  DEVICE_VENDOR := ZTE
+  DEVICE_MODEL := MF286C3
+  DEVICE_PACKAGES := kmod-mt7603 kmod-sdhci-mt7620 kmod-usb2 kmod-usb3 kmod-fs-nfs-v3 kmod-fs-nfs-v4 kmod-l2tp kmod-mt7615e \
+  kmod-mt7663-firmware-ap kmod-mt7663-firmware-sta -wpad-basic-mbedtls -uboot-envtools \
+  kmod-usb-ledtrig-usbport nand-utils kmod-i2c-core hostapd \
```

Comment:
Removed wpad-basic-mbedtls and add hostapd? Why?

Reply:
firstly, I took the example below from the same file as a basis, and secondly, without hostapd Wi-Fi does not rise automatically
![screenshot](https://github.com/openwrt/openwrt/assets/2909755/7ce92f13-96aa-4e11-baf3-9cd5d08152d8)

Reply:
So don't delete the wpad-basic-mbedtls. This is hostapd by another name.

Reply:
ok, resolved